README.md

```diff
@@ -1 +1,29 @@
 # hw01
+
+## Biography 
+
+My name is **Maggie Hu** and I am a second-year graduate student at *Harris*.
+
+My courses include:
+
+* Professional courses
+  * Microeconomics
+  * Political Economy
+  * Econometrics
+
+* Programming course
+  * Computing for Social Sciences 
+  
+Here is a picture of me:
+
+![Maggie](https://github.com/Maggie-106/hw01/blob/master/Maggie.png?raw=true)
```

Comment:
I have no issues viewing the picture but I did notice that it was quite large. This is something I am also guilty of doing but I found that by adding {width = 30%} after the image path it should decrease the size of the image.